features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -213,6 +213,16 @@ CREATE TABLE IF NOT EXISTS SP_APP (
 
 ALTER TABLE SP_APP ADD CONSTRAINT APPLICATION_NAME_CONSTRAINT UNIQUE(APP_NAME, TENANT_ID);
 
+CREATE TABLE IF NOT EXISTS SP_METADATA (
+            ID INTEGER AUTO_INCREMENT,
+            SP_ID INTEGER,
+            NAME VARCHAR(255) NOT NULL,
+            VALUE VARCHAR(255) NOT NULL,
+            DISPLAY_NAME VARCHAR(255),
+            PRIMARY KEY (ID),
+            UNIQUE (SP_ID, NAME),
```

Comment:
better add a name for the constraint, it will make easy to drop the constraint if needed.